src/components/Login.js

```diff
@@ -82,81 +78,80 @@ class Login extends React.Component {
-          <stop offset="1" stopColor={gradientColors[this.state.index][1]} stopOpacity="0.47" />
+          <stop stopColor={gradientColors[index][0]} />
+          <stop offset="1" stopColor={gradientColors[index][1]} stopOpacity="0.47" />
         </linearGradient>
       </defs>
     </svg>
   );
 
-  render = () => {
-    const textHighlightStyle = {
-      background: `linear-gradient(180deg, rgba(255,255,255,0) 80%, ${
-        gradientColors[this.state.index][0]
-      } 50%)`,
-    };
-    return (
-      <div className="login-page-content">
-        <div className="login-page-content-container">
-          <div
-            className="bottom-right-toggle"
-            onClick={() => this.setState({ index: Math.floor(Math.random() * 5) })}
-          >
-            <FontAwesomeIcon icon={faRedo} />
-          </div>
-          <div className="login-page-content-main">
-            <div>
-              <h1 className="font-weight-bold">
-                The ACM
-                {' '}
-                <span className="teachla-green">Teach LA</span>
-                {' '}
-                <span style={textHighlightStyle}>Online Editor</span>
-              </h1>
-              <p>a web IDE that lets you write and run Python &amp; Processing code, anywhere.</p>
-              {this.props.create ? (
-                <CreateUserForm
-                  initialState={this.props.initialState}
-                  themeColor={themeColors[this.state.index][0]}
-                  textColor={themeColors[this.state.index][2]}
-                />
-              ) : (
-                <LoginForm
-                  themeColor={themeColors[this.state.index][0]}
-                  textColor={themeColors[this.state.index][1]}
-                />
-              )}
-            </div>
-          </div>
-          <div className="login-page-content-footer">
-            <FontAwesomeIcon icon={faPaintBrush} />
-            {' '}
-            <FontAwesomeIcon icon={faCode} />
-            {' '}
-            <FontAwesomeIcon icon={faRocket} />
-            {' '}
-            by
-            {' '}
-            <a href="https://teachla.uclaacm.com" target="_blank" rel="noopener noreferrer">
-              <span className="teachla-green">ACM Teach LA</span>
-            </a>
-            {' '}
-            with
-            {' '}
-            <FontAwesomeIcon className="beating-heart" icon={faHeart} />
+  const textHighlightStyle = {
+    background: `linear-gradient(180deg, rgba(255,255,255,0) 80%, ${gradientColors[index][0]} 50%)`,
+  };
+  return (
+    <div className="login-page-content">
+      <div className="login-page-content-container">
+        <div
+          className="bottom-right-toggle"
+          onClick={() => setIndex(Math.floor(Math.random() * 5))}
+          onKeyDown={(e) => {
+            if (e.key === 'Tab') return;
+            setIndex(Math.floor(Math.random() * 5));
+          }}
```

Comment:
Might be worth making `setIndex(Math.floor(Math.random() * 5))` a one-liner `const` function!

Reply:
Hey Matt, have a look at the solution I have in place to dynamically render the SVG, it's looking quite nice when adjusting the window height.

Reply:
Ahh disregard, on larger screen heights the SVG appears buggy.  I'll have to take a closer look.

![Screen Shot 2021-11-07 at 8 47 56 PM](https://user-images.githubusercontent.com/18103497/140672004-926ab429-189c-4433-94d6-867de9c67158.png)

Reply:
Ok I fixed this bug.  If the SVG was being rendered and then the innerHeight was increased, the SVG would look manipulated because it was not growing to fit the innerHeight.  Now the svg is growing with innerHeight.